pkg/chunk/cache/redis_cache_test.go

```diff
@@ -79,12 +50,19 @@ func TestRedisCache(t *testing.T) {
 	}
 }
 
-func mockRedisStore(conn *redigomock.Conn, keys []string, bufs [][]byte) {
-	conn.Command("MULTI")
-	ret := []interface{}{}
-	for i := range keys {
-		conn.Command("SETEX", keys[i], 0, bufs[i])
-		ret = append(ret, "OK")
+func mockRedisCache() (*RedisCache, error) {
+	redisServer, err := miniredis.Run()
+	if err != nil {
+		return nil, err
+
+	}
+
+	redisClient := &redisBasicClient{
+		expiration: time.Minute,
+		timeout:    10 * time.Millisecond,
```

Comment:
We could increase it to 100ms, just to avoid the risk of having flaky tests on a slow CI run.